maps/Z.03.USS_Almayer.dmm

```diff
@@ -478,7 +478,7 @@
 "ajj" = (/obj/machinery/alarm/almayer{dir = 1},/turf/open/floor/grass,/area/almayer/living/starboard_garden)
 "ajk" = (/obj/structure/cable{d1 = 4; d2 = 8; icon_state = "4-8"; pixel_x = 0},/turf/open/floor/almayer{tag = "icon-bluecorner (EAST)"; icon_state = "bluecorner"; dir = 4},/area/almayer/hallways/aft_hallway)
 "ajl" = (/turf/closed/wall/almayer/white,/area/almayer/medical/upper_medical)
-"ajm" = (/obj/machinery/door/airlock/multi_tile/almayer/medidoor{name = "\improper Medical Bay"; req_access_txt = "20"; req_one_access_txt = "0"},/obj/machinery/door/firedoor/border_only/almayer{tag = "icon-door_open"; icon_state = "door_open"; dir = 2},/turf/open/floor/almayer{icon_state = "dark_sterile"; tag = "icon-sterile"},/area/almayer/medical/upper_medical)
+"ajm" = (/obj/machinery/door/airlock/multi_tile/almayer/medidoor{name = "\improper Medical Bay"; req_access_txt = "14"; req_one_access_txt = "0"},/obj/machinery/door/firedoor/border_only/almayer{tag = "icon-door_open"; icon_state = "door_open"; dir = 2},/turf/open/floor/almayer{icon_state = "dark_sterile"; tag = "icon-sterile"},/area/almayer/medical/upper_medical)

```

Comment:
Don't use `req_access_txt`, this is why we can't have nice things.
Also, alter `/obj/machinery/door/airlock/multi_tile/almayer/medidoor` or makes child of them to be placed in the map instead of editing these things in the map.